pkg/csi/service/wcp/controller_helper.go

```diff
@@ -542,7 +542,7 @@ func (c *controller) GetVolumeToHostMapping(ctx context.Context) (map[string]str
 	}
 
 	// Get all the hosts belonging to the cluster
-	hostSystems, err := vc.GetHostsByCluster(ctx, c.manager.CnsConfig.Global.ClusterID)
+	hostSystems, err := vc.GetHostsByCluster(ctx, c.manager.CnsConfig.Global.SupervisorID)
```

Comment:
here `c.manager.CnsConfig.Global.SupervisorID` is not a cluster MoRef. so revert it back.
and disable the list volume API on the Streched supervisor cluster, as we do not support Pod VM on Streched supervisor cluster.

Reply:
done